src/ontology/uberon-edit.obo

```diff
@@ -67,6 +67,7 @@ import: http://purl.obolibrary.org/obo/uberon/imports/ncbitaxon_import.owl
 import: http://purl.obolibrary.org/obo/uberon/imports/pato_import.owl
 import: http://purl.obolibrary.org/obo/uberon/imports/pr_import.owl
 import: http://purl.obolibrary.org/obo/uberon/imports/ro_import.owl
+import: http://purl.obolibrary.org/obo/uberon/imports/bspo_import.owl
```

Comment:
whoops, not sure if we still want this in, was trying to use upper from BSPO, but its a class and not an object property so no idea how to use it